conductr_cli/conduct.py

```diff
@@ -109,24 +110,26 @@ def build_parser():
     # Sub-parser for `events` sub-command
     events_parser = subparsers.add_parser('events',
                                           help='show bundle events')
-    events_parser.add_argument('--service',
-                               default='http://{}:9210'.format(default_ip),
-                               help='Events service address')
+    add_ip_and_port(events_parser)
```

Comment:
This is consistent with the rest of the commands where we have `--ip` switch.

Since the `conduct events` and `conduct logs` are talking to ConductR directly now, I think it makes sense to consolidate them.

Do let me know if you want everything to be updated to `--service` instead though.